tests/utils/test_file_utils.py

```diff
@@ -344,3 +345,23 @@ def test_handle_readonly_on_windows(tmpdir):
 )
 def test_local_file_uri_to_path_on_windows(input_uri, expected_path):
     assert local_file_uri_to_path(input_uri) == expected_path
+
+
+def test_shutil_copytree_without_file_permissions(tmpdir):
```

Comment:
```suggestion
def test_shutil_copytree_without_file_permissions(tmp_path):
```

Let's use tmp_path.

Reply:
yep! Changed :) I'll leave the rest of those tests in that suite for another PR ;)